script.module.resolveurl/lib/resolveurl/plugins/mixdrop.py

```diff
@@ -29,8 +29,8 @@ class MixDropResolver(ResolveUrl):
                'mixdroop.co', 'mixdrop.vc', 'mixdrop.ag', 'mdy48tn97.com',
                'md3b0j6hj.com', 'mdbekjwqa.pw', 'mdfx9dc8n.net', 'mixdropjmk.pw',
                'mixdrop21.net', 'mixdrop.is', 'mixdrop.si', 'mixdrop23.net', 'mixdrop.nu',
-               'mixdrop.ms', 'mdzsmutpcvykb.net', 'mixdrop.ps']
-    pattern = r'(?://|\.)((?:mixdro*p\d*(?:jmk)?|md(?:3b0j6hj|bekjwqa|fx9dc8n|y48tn97|zsmutpcvykb))\.' \
+               'mixdrop.ms', 'mdzsmutpcvykb.net', 'mixdrop.ps', 'mxdrop.to']
+    pattern = r'(?://|\.)((?:m[i]?xdro*p\d*(?:jmk)?|md(?:3b0j6hj|bekjwqa|fx9dc8n|y48tn97|zsmutpcvykb))\.' \
```

Comment:
Dont need `[ ]` when there is only one character. Can be `mi?xdro*p`. Please update your PR